xsd/xsd.go

```diff
@@ -52,6 +52,8 @@ type Element struct {
 	// True if this element can have any name. See
 	// http://www.w3.org/TR/2004/REC-xmlschema-1-20041028/structures.html#element-any
 	Wildcard bool
+	// limit any type by namespace
+	LimitedByNamespace string
```

Comment:
Keep same tense as rest of fields and comment?
```suggestion
	LimitByNamespace string
```

Reply:
updated